src/simmate/apps/configs.py

```diff
@@ -17,3 +17,11 @@ class VaspConfig(AppConfig):
 
 class EvolutionConfig(AppConfig):
     name = "simmate.apps.evolution"
+
+
+class WarrenConfig(AppConfig):
+    name = "simmate.apps.warrenapp"
```

Comment:
throughout the PR, you jump between `warrenapp`, `warren`, and `warrenlab`. I would pick one and stick with it. 

I would vote for `warren_lab` because it follows the "standards" I kinda set elsewhere in the code

Reply:
For class names, this would be `WarrenLab` + `WarrenLabConfig` etc.

Reply:
Yep. warren_lab sounds good to me.